pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -410,16 +410,16 @@ providers:
 				managedConfig:  &corev1.ConfigMap{},
 				monitoringSharedConfig: &corev1.ConfigMap{
 					Data: map[string]string{
-						"alertmanagerURL": "https://alertmanager.url.com",
-						"grafanaURL":      "https://grafana.url.com",
-						"prometheusURL":   "https://prometheus.url.com",
-						"thanosURL":       "https://thanos.url.com",
+						"alertmanagerPublicURL": "https://alertmanager.url.com",
+						"grafanaPublicURL":      "https://grafana.url.com",
+						"prometheusPublicURL":   "https://prometheus.url.com",
+						"thanosPublicURL":       "https://thanos.url.com",
```

Comment:
Are the other teams making this change to the config maps they own?

Reply:
yes, both team shall do so